src/main/java/cms/gov/madie/measure/services/MeasureSetService.java

```diff
@@ -79,6 +81,29 @@ public MeasureSet updateOwnership(String measureSetId, String userId) {
     }
   }
 
+  public MeasureSet createCmsId(String measureSetId, String sequenceName, String username) {
+    Optional<MeasureSet> measureSet = measureSetRepository.findByMeasureSetId(measureSetId);
+    if (!measureSet.isPresent()) {
+      throw new InvalidMeasureSetIdException(
+          "No measure set exists for measure with measure set id " + measureSetId);
+    }
+    if (measureSet.get().getCmsId() > 0) {
+      throw new InvalidRequestException(
```

Comment:
we could reuse existing `ResourceNotFoundException`

Reply:
thanks, I changed it